src/include/daos_srv/container.h

```diff
@@ -72,17 +73,22 @@ struct ds_cont_child {
 				 sc_stopping:1;
 	uint32_t		 sc_dtx_flush_wait_count;
 
+	daos_epoch_t		 sc_rebuild_fence;
 	/* Aggregate ULT */
 	struct dss_sleep_ult	 *sc_agg_ult;
+
 	/*
-	 * Lower bound of aggregation epoch, it can be:
-	 *
-	 * < DAOS_EOPCH_MAX	: Some snapshot was deleted since last
-	 *			  round of aggregation
-	 * DAOS_EPOCH_MAX	: No snapshot deletion since last round of
-	 *			  aggregation
+	 * Snapshot/rebuild change HLC (0 means no change), which is used
+	 * to compare with the aggregation HLC, so it knows whether the
+	 * aggregation needs to be restart from 0.
 	 */
-	uint64_t		 sc_aggregation_min;
+	uint64_t		sc_snapshot_change_hlc;
+
+	/* HLC when the aggregation start, if it is smaller than
+	 * snapshot_change_hlc, then aggregation needs to restart from 0.
+	 */
+	uint64_t		sc_aggregation_start_hlc;
```

Comment:
could you revise the comment a bit? I think it's not about aggregation start or restart, it's about the hlc of last full aggregation scan, so the name could be 'sc_aggregate_full_hlc'?